sdk/servicebus/microsoft-azure-servicebus/pom.xml

```diff
@@ -78,6 +78,12 @@
 	    <groupId>com.google.code.gson</groupId>
 	    <artifactId>gson</artifactId>
 	</dependency>
```

Comment:
Do we need this dependency now? You can remove it.

Reply:
SJ requested me to use this dependency instead of our own implementation, maybe we can talk with him once he is back

---
In reply to: [303117056](https://github.com/Azure/azure-sdk-for-java/pull/4004#discussion_r303117056) [](ancestors = 303117056)